docker/Dockerfile

```diff
@@ -93,3 +93,6 @@ ENV NODE_ENV=production
 
 ENTRYPOINT ["pm2-runtime", "bin/iotagent-ul"]
 CMD ["-- ", "config.js"]
+
+# Expose 4041 for NORTH PORT, 7896 for HTTP PORT
+EXPOSE 4041 7896
```

Comment:
Don't hardcode the ports, use default values and existing `ENV`s:

```Dockerfile
# Expose IOTA_NORTH_PORT and  IOTA_HTTP_PORT
EXPOSE ${IOTA_NORTH_PORT:-4041} ${IOTA_HTTP_PORT:-7896}
```

Reply:
Fully agree. @Jagatjot note that we have env vars for this already documented in https://hub.docker.com/r/telefonicaiot/iotagent-ul/

```
    environment:
       ...
        - "IOTA_NORTH_PORT=4041"
        - "IOTA_HTTP_PORT=7896"
       ...
```

In addition, note that ENTRYPOINT and CMD should be the last lines in the file (is part of our style in Dockerfiles ;) so please move the EXPOSE clause up. Maybe just after `ENV NODE_ENV=production`  is a good place. @jason-fox maybe have some feedback on this.

Reply:
Fixed by https://github.com/telefonicaid/iotagent-ul/pull/342/commits/2cbfd93109345a6800f8f8877ea2f0c76167e2fd

Reply:
LGTM